_glossary/conditional/user-is-logged-in-start.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: side-nav
+title: User Is logged In Start
+last-updated: 01-03-2013
+nav-position: 17
+parent: conditional
+tag: <!--WDK:isloggedin:start-->
```

Comment:
'End' already exists and you've add the deprecated flag already... https://github.com/createdotnet/wdk/blob/v1.1/_glossary/conditional/user-is-logged-in-end.md

I wasn't going to add the flag here in case it goes live before the v1.1 branch does. I could merge them together and add it?